pycontw/pycontw-frontend
# feat(speech-detail): add the section of related speeches list

## Types of changes

* **New feature**

## Description

update the detail page, and add relatedCardCollection component

## Steps to Test This Pull Request

1. Open /conference/talks.
2. Click any detail page, there's a related speeches zone.
3. Click the arrow to slide related card, and check the hyperlink, and RWD.

![Aaron Swartz](https://raw.githubusercontent.com/furecool/markdown-img/master/pycon/detail-related.gif)

## Related Issue

#246

## Additional context
